content/en/continuous-deployment/release-notes/rn-armory-spinnaker/armoryspinnaker_v2-30-0.md

```diff
@@ -0,0 +1,1657 @@
+
+Red/Black to Blue/Green migration details:
+* Front50 already supports migrations but you need to be enable it by adding migrations.enabled=true in your Front50 config. When Front50 starts, it automatically migrates existing pipelines using red/black to blue/green.
+* *If you do not enable migration*, red/black continues to work until the Spinnaker community decides to remove red/black.
+
+### Artifact Handling
+Changes to the way artifact constraints on triggers work
+If you have a pipeline with multiple triggers using different artifact constraints/expected artifacts, these have for a while been evaluated in an unexpected matter. To learn more, visit 
+https://spinnaker.io/changelogs/1.30.0-changelog/#changes-to-the-way-artifact-constraints-on-triggers-work.
+
+### Spring Boot 2.4 changes
+* Read more about Spring boot in the [1.30 Release notes](https://spinnaker.io/changelogs/1.30.0-changelog/#spring-boot-24)
+
+###  Spinnaker Community Contributions
+
+There have also been numerous enhancements, fixes, and features across all of Spinnaker's other services. See the
+[Spinnaker v1.30.2](https://www.spinnaker.io/changelogs/1.30.2-changelog/) changelog for details.
+
+
+### Bill Of Materials (BOM)
+
+Here's the BOM for this version.
+<details><summary>Expand</summary>
+<pre class="highlight">
+<code>artifactSources:
+  dockerRegistry: docker.io/armory
+dependencies:
+  redis:
+    commit: null
+    version: 2:2.8.4-2
+services:
+  clouddriver:
+    commit: 5dfa9fe9e2285a875f0f39cb29e46a6470f34665
+    version: 2.30.0
+  deck:
+    commit: a5ae63596f79df5c3dd4999253a9ed72dece7de3
+    version: 2.30.0
+  dinghy:
+    commit: 5250de80948732c8caac6ffc5293a8af80a63a0f
+    version: 2.30.0
+  echo:
+    commit: 86c586f7c523a4c320189eff00731271b8d31e32
+    version: 2.30.0
+  fiat:
+    commit: 0150c145b239568c294ab88251dc2fbb20ace279
+    version: 2.30.0
+  front50:
+    commit: bdbf36960c30b3599bdf0fa31620dfaf08927074
+    version: 2.30.0
+  gate:
+    commit: 679225a36b20fe39ecb175813929972c497d1a92
+    version: 2.30.0
+  igor:
+    commit: 020e01bbeaadf3d5eb745b33180bd1011c4b068f
+    version: 2.30.0
+  kayenta:
+    commit: 16aa401c453de95b670524b491cbaa682bcf2817
+    version: 2.30.0
+  monitoring-daemon:
+    commit: null
+    version: 2.26.0
+  monitoring-third-party:
+    commit: null
+    version: 2.26.0
+  orca:
+    commit: 25ccf32473a809846c1c3d4c967bb05ad9622549
+    version: 2.30.0
+  rosco:
+    commit: c72a24d8c560ea7b27fd8ecf45c9c11ca63682f4
+    version: 2.30.0
+  terraformer:
+    commit: 418546f57129380e383e62b6178ed582e6d64a93
+    version: 2.30.0
+timestamp: "2023-05-31 22:11:31"
+version: 2.30.0
+</code>
+</pre>
+</details>
+
+### Armory
+
+
+#### Armory Clouddriver - 2.28.0...2.30.0
+
+  - chore(cd): update base service version to clouddriver:2022.07.07.23.26.37.release-1.28.x (#688)
```

Comment:
This is a suggestion for future releases rather than 2.30.0. As an end user these commits don't tell me anything helpful and can be ommitted without lost of clarity. @link108 is there already a ticket in our backlog to track removal of these autobump messages from the release note automation?

Reply:
yup https://armory.atlassian.net/browse/DYN-343